media-libs/libiptcdata/libiptcdata-1.0.5.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{3_7,3_8} )
+
+inherit eutils python-single-r1
+
+DESCRIPTION="library for International Press Telecommunications Council (IPTC) metadata"
+HOMEPAGE="https://github.com/ianw/libiptcdata http://libiptcdata.sourceforge.net"
+SRC_URI="https://github.com/ianw/${PN}/releases/download/release_1_0_5/${P}.tar.gz"
+
+LICENSE="LGPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="doc examples nls python"
+
+RDEPEND="python? ( ${PYTHON_DEPS} )
+	nls? ( virtual/libintl )"
+DEPEND="${RDEPEND}
+	nls? ( >=sys-devel/gettext-0.13.1 )
+	doc? ( >=dev-util/gtk-doc-1 )"
```

Comment:
Please keep them sorted. At least gettext will probably be a `BDEPEND` instead, don't know about gtk-doc though (maybe look for examples using EAPI-7 and gtk-doc).

Reply:
yes, both are BDEPEND